lib/engine/game/g_1825/meta.rb

```diff
@@ -175,6 +181,8 @@ def self.check_options(options, num_players)
           regionals[2] = true if optional_rules.include?(:r2)
           regionals[3] = true if optional_rules.include?(:r3)
 
+          dbvariants[2] = true if optional_rules.include?(:db2)
```

Comment:
Is dbvariants ever used? Why do you need this?

Reply:
My mistake. I started thinking of a scheme like regionals[] etc but the Dave Berry variants don't really have much in common other than the author.